src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/scriptrunner/scriptrunner.py

```diff
@@ -26,27 +26,23 @@ def _main(settings, optlist, args):
             xunitrunner
                 [--config config.json]
                 [--setting name=value]
-                --script
-                [--args arg1 arg2...]
+                --script=path
+                [args]
         """
-        optdict = dict(optlist)
         log.info("BuildTools Helix Script Runner v0.1 starting")
-        if '--args' in optdict:
-            script_arguments = optdict['--args']
-            log.info("Script Arguments:"+script_arguments)
```

Comment:
any special reason to delete a useful log statement?

Reply:
Thanks Matt. My intent was to simplify the code by removing an if+variable that was only used for logging. I had then changed the code to dump the script_arguments in another place; However, after the recent changes I had also stripped that off. I'll move the log back.